src/components/molecules/SpaceEditFormNG/SpaceEditFormNG.tsx

```diff
@@ -223,10 +223,16 @@ export const SpaceEditFormNG: React.FC<SpaceEditFormNGProps> = ({
 
   const { ownedVenues } = useOwnedVenues({});
 
-  const venueSpaces = ownedVenues.map(({ name, template }) => ({
-    name,
-    template,
-  }));
+  const backButtonOptionList = ownedVenues.filter(({ id, name, template }) => {
```

Comment:
[NOTE] Annoyance, having to write the same code twice and check for minor differences.